src/test/java/uk/gov/digital/ho/hocs/audit/aws/listeners/integration/BaseAwsSqsIntegrationTest.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.digital.ho.hocs.audit.aws.listeners.integration;
+
+import com.amazonaws.services.sqs.AmazonSQSAsync;
+import com.amazonaws.services.sqs.model.PurgeQueueRequest;
+import org.junit.After;
+import org.junit.Before;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.test.context.ActiveProfiles;
+
+import java.util.List;
+
+@ActiveProfiles("local")
+public class BaseAwsSqsIntegrationTest {
+
+    private static final String APPROXIMATE_NUMBER_OF_MESSAGES = "ApproximateNumberOfMessages";
+    private static final String APPROXIMATE_NUMBER_OF_MESSAGES_NOT_VISIBLE = "ApproximateNumberOfMessagesNotVisible";
```

Comment:
I don't understand the rational behind this, it seems the binary opposite of the line above, we're using the awaitility library because of the visibility timeout issues so we leave that library to handle that.

Reply:
As we're not testing the DLQ, when a message fails it gets flagged visible: false flag. This causes this attribute to go to 1 before it gets sent to the DLQ.